src/articles/FindArticles.js

```diff
@@ -88,6 +84,15 @@ export default function NewArticles() {
       });
     }
     document.title = "Zeeguu";
+
+    const timeOutRequestId = setTimeout(() => {
```

Comment:
So `setTimeout` sets a timer to execute a function when the timeout expires. 

Would it be fair to name that function something like: `setExtensionAbsentIfNoMessageReceived`? 

In that case, maybe the timer should be named something like:  `timerToSetExtensionAbsentIfNoMessageReceived` or similar?

Reply:
I just realized, I might be able to do that (setting the timer and handling it) inside the checkExtensionInstalled, which might make it easier to read.
I don't like using very long variable names personally, so I would suggest maybe using a comment on the method to explain the messaging system rather than giving it a very long descriptive name. But if you think that's better I can change them into a name like that. 

But first I will test the Hook + Moving it all inside the checkExtensionInstalled method.

Reply:
I don't necessarily like long names either :)

And I think if you succeed in scoping the code all in one file, the name can be more concise as long as it's still expressing exactly what that variable represents :) But with complex behavior split across files I had to give it that name to make sense of the code myself :)